app/channels/room_channel.rb

```diff
@@ -22,43 +22,56 @@ def speak(data)
     if client_action["join"] || client_action["add-ai-player"]
       player = client_action["add-ai-player"] || current_user
       @game.add_player(player)
-      # start_game(@game) if ActionCable.server.connections.map(&:current_user)
-      #   .count == @game.users.count
     elsif client_action["start-game"]
       start_game(@game)
     elsif @game.started
+      @game = Game.find(@game.id)
```

Comment:
👍 That's a great idea. I was having issues with getting information that was just saved to the database--I'm not sure if it has to do with the way things are being cached behind the scenes or what. the @game.reload sounds like a better solution. Thanks!